features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -502,6 +544,21 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     }, [ selectedGrantTypes, isGrantChanged ]);
 
+    // useEffect(() => {
+    //     setHybridFlowResponseTypeField(false);
+    //     if (selectedGrantTypes?.includes(ApplicationManagementConstants.AUTHORIZATION_CODE_GRANT)) {
+    //         setHybridFlowResponseTypeField(true);
+    //     }
+    // }, [ selectedGrantTypes, isGrantChanged ]);
+
+    useEffect(() => {
+        setHybridFlowEnableConfig(false);
+        if (selectedGrantTypes?.includes(ApplicationManagementConstants.AUTHORIZATION_CODE_GRANT)) {
+            setHybridFlowEnableConfig(true);
+        }
+    }, [ selectedGrantTypes, isGrantChanged ]);
```

Comment:
Does this useEffect need to be triggered on changing isGrantChanged?

Reply:
With the current IS implementation hybrid flow can be used only when authorization code grant is enabled, Due to that we're listening the grant type change to show the hybrid flow response type config.